androidsdk/src/main/java/com/wootric/androidsdk/objects/WootricCustomThankYou.java

```diff
@@ -49,8 +49,8 @@
     private boolean scoreInUrl;
     private boolean commentInUrl;
 
-    public String getTextForScore(int scoreValue) {
-        Score score = new Score(scoreValue);
+    public String getTextForScore(int scoreValue, String surveyType) {
```

Comment:
This gets passed on every method, why don't you add it to the constructor instead?